htmd/parameterization/fftype.py

```diff
@@ -3,31 +3,52 @@
 # Distributed under HTMD Software License Agreement
 # No redistribution in whole or part
 #
-
-import shutil
+import logging
 import subprocess
 import os
+import re
+import unittest
 from tempfile import TemporaryDirectory
-from htmd.parameterization.readers import readPREPI, readFRCMOD, readRTF
+
 import numpy as np
-import parmed
-import logging
-import unittest
 
 logger = logging.getLogger(__name__)
 
 
 fftypemethods = ('CGenFF_2b6', 'GAFF', 'GAFF2')
 
 
-def listFftypemethods():
```

Comment:
I continue to see no reason to delete this, but ok

Reply:
What is the reason to keep it?